src/oas3/transformers/parameters.ts

```diff
@@ -20,6 +23,7 @@ export const translateToSharedParameters = withContext<
     query: [],
     cookie: [],
     path: [],
+    unknownParameters: [],
```

Comment:
IMHO this should be called `unknown` like we have it in the `request`. It serves a very similar use case.
We could also store other objects there, if happened to be needed in future for some different spec

Reply:
we're being explicit here so that on the other side we can handle it properly.